plugins/clickhouse/include/easy_client.hpp

```diff
@@ -0,0 +1,43 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2025 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include "clickhouse/client.h"
```

Comment:
```suggestion
#include <clickhouse/client.h>
```

This is also the case in many other files, please double-check this.